file-formats/apic/examples/z_api_catalog_example.apic.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Is this the expected example filename? I think it should be `z_aff_example_apic.apic.json`

Please consider the name and link of the file in the README.md file

Reply:
Should be solved